spyderlib/config.py

```diff
@@ -512,17 +514,45 @@ def is_ubuntu():
               'editor/toggle comment': "Ctrl+1",
               'editor/blockcomment': "Ctrl+4",
               'editor/unblockcomment': "Ctrl+5",
+              'editor/start of line': "Meta+A",
+              'editor/end of line': "Meta+E",
+              'editor/previous line': "Meta+P",
+              'editor/next line': "Meta+N",
+              'editor/previous char': "Meta+B",
+              'editor/next char': "Meta+F",
+              'editor/previous word': "Meta+Left",
+              'editor/next word': "Meta+Right",
+              'editor/kill to line end': "Meta+K",
+              'editor/kill to line start': "Meta+U",
+              'editor/yank': 'Meta+Y',
+              'editor/rotate kill ring': 'Shift+Meta+Y',
+              'editor/kill previous word': 'Meta+Backspace',
+              'editor/kill next word': 'Meta+D',
+              'editor/start of document': 'Meta+Less' if DARWIN else \
+                                          'Ctrl+Up',
+              'editor/end of document': 'Meta+Greater' if DARWIN else \
+                                        'Ctrl+Down',
               # -- In widgets/editor
               'editor/inspect current object': 'Ctrl+I',
               'editor/go to line': 'Ctrl+L',
               'editor/file list management': 'Ctrl+E',
               'editor/go to previous file': 'Ctrl+Tab',
               'editor/go to next file': 'Ctrl+Shift+Tab',
+              'editor/zoom in': 'Ctrl+Plus',
+              'editor/zoom out': 'Ctrl+Minus',
+              'editor/zoom reset': 'Ctrl+0',

```

Comment:
Yes, please :-) Given that the IPython shortcuts are not configurable, I think it's better to have only one way to do zooms, instead of a customized one for the Editor and a fixed one for qtconsole.